### PR TITLE
chore(flags): remove old feature flags 

### LINKED ISSUE #2995: Remove old feature flags
### Overview

We want to remove the feature flags for

* `OnRamperFiatRamp`
* `MtPelerinFiatRamp`
* `RainbowCharts`

### References and additional details

* An example of how to do this https://github.com/shapeshift/web/pull/2861

### Acceptance Criteria

* There are no remnants or dead code associated with these flags, including environment variables
* All of these features still work as they currently do

### Need By Date

n/a

### Screenshots/Mockups

image

### Ownership

- [X] If my bounty needs engineering or needs product I have added the respective labels on the right
- [X] As the sponsor of this bounty I will review the changes in a preview environment (ops/product) or review the PR (engineering)

### Estimated effort

2 hours

### Sponsor / Stakeholder

0xdef1cafe

### Bounty Hunters

- [Join our discord](https://discord.gg/shapeshift)
- Include an expected timeline for you to complete work in the work plan when you apply for this bounty!
- Please refer to [this link](https://shapeshiftdao.gitbook.io/getting-started/how-to-contribute/bounties#resources-for-new-contributors-and-bounty-hunters) for some basic info
- Please ***do not*** start work on this issue until you are approved in Gitcoin.

### PR DESCRIPTION
**Pull request by developerfred**
## Description

remove flags:
- [x] OnRamperFiatRamp 
- [x] MtPelerinFiatRamp
- [x] RainbowCharts

## Notice

- [x] Have you followed the guidelines in our [Contributing]('https://github.com/shapeshift/web/blob/develop/CONTRIBUTING.md) guide?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/shapeshift/web/pulls) for the same update/change?

## Pull Request Type

- [ ] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [x] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [ ] :nail_care: New Feature (Breaking/Non-breaking Change)

## Issue (if applicable)

closes #2995

## Risk

no risk 

## Testing

- [x] ``yarn lint ``
- [x] build test

### CODE REVIEW

File: src/pages/Dashboard/Portfolio.tsx [hunk 1]
```diff
@@ -44,7 +44,6 @@ export const Portfolio = () => {
   const loading = useSelector(selectPortfolioLoading)
   const isLoaded = !loading
 
-  const isRainbowChartsEnabled = useFeatureFlag('RainbowCharts')
```
Comment (0xApotheosis) [inline src/pages/Dashboard/Portfolio.tsx:47]: `totalBalancePlusLpHoldings` above needs to go, too.

**Review by 0xApotheosis — CHANGES_REQUESTED** [review]
Thanks for the PR @developerfred.

This PR has a lot of lint violations. Please ensure lint passes before opening. 
Also ensure `yarn dev` runs without error.

Run `yarn lint` to see current issues, and `yarn lint:fix` should take care of most.

**Comment by 0xApotheosis** [comment]
Converting to draft until this PR is ready.

**Comment by developerfred** [comment]
I'll run the lint and fix the errors

**Comment by 0xApotheosis** [comment]
Also @developerfred please ensure the template is filled out before opening.

Closing this PR until the contribution guidelines have been followed.

File: src/config.ts [hunk 2]
```diff
@@ -64,9 +64,7 @@ const validators = {
   REACT_APP_FEATURE_COWSWAP: bool({ default: false }),
   REACT_APP_FEATURE_YAT: bool({ default: false }),
   REACT_APP_FEATURE_AXELAR: bool({ default: false }),
-  REACT_APP_FEATURE_RAINBOW_CHARTS: bool({ default: false }),
   REACT_APP_FEATURE_MTPELERIN_FIAT_RAMP: bool({ default: false }),
```
[the remark this answers is not in the document]
Comment (developerfred) [reply src/config.ts:67]: done, thx!

File: src/pages/Dashboard/Portfolio.tsx (hunk 1 above)
Comment (developerfred) [reply src/pages/Dashboard/Portfolio.tsx:47]: done, thx!

**Comment by developerfred** [comment]
@0xApotheosis done here https://github.com/developerfred/shapeshift-web/commit/b905b9c8a98e63034f7f2a7e91be5a552741b32c

**Comment by 0xApotheosis** [comment]
@developerfred still no PR template?

**Comment by 0xdef1cafe** [comment]
@developerfred please do not apply for future bounties.

**Comment by developerfred** [comment]
@0xdef1cafe ok, thanks! sorry about this.

**Pull request closed without merging**